posthog/models/web_preaggregated/team_selection_strategies.py

```diff
@@ -75,36 +72,31 @@ def get_teams(self, context: dagster.OpExecutionContext) -> set[int]:
             return set()
 
 
-class FeatureEnrollmentStrategy(TeamSelectionStrategy):
-    """Select teams where users have enrolled in a specific feature preview (default: web-analytics-api)."""
+class ProjectSettingsStrategy(TeamSelectionStrategy):
```

Comment:
I avoided changing the logic here too much, so we can move fast, but with more time, we could make this the default way now to control if the team/org has the tables or not by making the other strategies update the project setting and use it for everything.